frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -218,48 +223,70 @@ const TranslationPage = () => {
       </Box>
     ) : null;
   };
-
   return (
-    <Box margin="20px 20px 0px 20px">
-      <Text size="lg">Story Title Here</Text>
-      <Text as="ins">View story details</Text>
-      <FontSizeSlider setFontSize={handleFontSizeChange} />
-      <Flex>
-        <Flex direction="column" width="75vw">
-          <Flex direction="row">
-            <IconButton
-              variant="ghost"
-              aria-label="Undo Change"
-              onClick={undoChange}
-              icon={<Icon as={MdUndo} />}
-            />
-            <IconButton
-              variant="ghost"
-              aria-label="Redo Change"
-              onClick={redoChange}
-              icon={<Icon as={MdRedo} />}
-            />
+    <Box height="100vh" overflow="hidden">
+      <Header title={title} />
+      <Divider />
+      <Flex justify="space-between">
+        <Box minWidth="calc(100vw - 350px)" maxWidth="75vw">
```

Comment:
woah thats some spooky math - can you describe what's going on?

Reply:
realized this was unnecessary, removed!